src/test/test-install.c

```diff
@@ -35,8 +38,10 @@ int main(int argc, char* argv[]) {
 
         HASHMAP_FOREACH(p, h) {
                 UnitFileState s = _UNIT_FILE_STATE_INVALID;
+                _cleanup_free_ char *unit_filename = NULL;
 
-                r = unit_file_get_state(RUNTIME_SCOPE_SYSTEM, NULL, basename(p->path), &s);
+                assert_se(path_extract_filename(p->path, &unit_filename) >= 0);
```

Comment:
ASSERT_OK

(here and everywhere appropriate)

Reply:
Sure.  Do I need to change all the tests I have been touching, or only the tests I added/changed?

Reply:
Only lines you modify. If you feel like it you can do more but don't feel like you have to migrate stuff you aren't touching

Reply:
Just the lines you touch is OK.

Reply:
I ended up doing more than the lines I touched, but rather the entire "units" around them.  Hopefully this change is okay.